recipe/meta.yaml

```
@@ -11,7 +11,7 @@ source:
   sha256: 469644ac7d1d37ab82b7213d3aae90820c3d840aa483967b006554501fe725ac
 
 build:
-  number: 2
+  number: 3
   skip: true  # [(not linux) or cuda_compiler_version != "11.8"]
   ignore_run_exports_from:
     - {{ compiler("cuda") }}
```

This should override [the upstream `CUDA_HOME` logic]( https://github.com/rapidsai/ptxcompiler/blob/ae91f8dad5e5ff7103128eaf1a77b923887334ec/setup.py#L25-L40 ) to pick up the appropriate `targets` directory

```suggestion
    - {{ compiler("cuda") }}
  script_env:
    - CUDA_HOME=/usr/local/cuda/targets/sbsa-linux     # [aarch64]
    - CUDA_HOME=/usr/local/cuda/targets/ppc64le-linux  # [ppc64le]
```